metricbeat/module/coredns/stats/stats_test.go

```diff
@@ -0,0 +1,37 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package stats
+
+import (
+	"testing"
+
+	"github.com/elastic/beats/metricbeat/helper/prometheus/ptest"
+)
+
+const testFile = "_meta/test/metrics"
+
+func TestEventMapping(t *testing.T) {
```

Comment:
This will also not be needed anymore if you have `testdata`. It will also validate if all the fields are documented.